app.js

```diff
@@ -6,20 +6,36 @@
 // a number of minutes as an argument and returns
 // the number of seconds
 
-function convertToSeconds() {}
+function convertToSeconds(x) {
+    return x * 60;
+}
 
 console.log(`convertToSeconds(10) === 600 : ${convertToSeconds(10) === 600}`);
 
 // 2. areaOfTriangle
 // Write a function called areaOfTriangle that takes in the base and height
 // of a triangle and returns its area
-function areaOfTriangle() {}
+function areaOfTriangle(x, y) {
+    return x + (2 * y) ;
```

Comment:
this is technically accurate for a "true" result with the below console log, but to find the area for all triangles you would need a version of "return (x * y) / 2 "